streams/bluetooth.py

```diff
@@ -0,0 +1,159 @@
+#! /usr/bin/python3
+# -*- coding: utf-8 -*-
+
+""" Bluetooth currently only supports at most one Bluetooth stream. Handles media controls and audio. """
+
+import time
+import json
+import argparse
+import sys
+from bluezero import dbus_tools
+from bluezero import media_player
+
+parser = argparse.ArgumentParser(prog='bluetooth', description='track bluetooth metadata and control playback')
+parser.add_argument('--command', type=str, help='playback command (play, pause, next, previous)', nargs='?',
+                    default=None)
+parser.add_argument('--song-info', type=str, help='file to update with current song information in json format')
+parser.add_argument('--log', type=str, help='log file (defaults to stdout)', default=None)
+parser.add_argument('--verbose', action='store_true', help='show more verbose output')
+args = parser.parse_args()
+
+def log(info):
+  if args.log:
+    try:
+      with open(args.log, 'a') as f:
+        print(info, file=f)
+    except:
+      print(f'Error writing to logfile: {args.log}')
+      print(info)
+  else:
+    print(info)
+
+if args.song_info:
```

Comment:
What is this?

Reply:
I don't know. It doesn't do anything right now, should I remove it?

Reply:
I removed it, was probably some debug prints.